spec/MC_ffg.tla

```diff
@@ -120,6 +129,10 @@ Next == UNCHANGED single_node_state
 
 NoSlashableInv == get_slashable_nodes(single_node_state.view_votes) = {}
 
+EbbAndFLowInv == 
```

Comment:
Actually, I don't think `EbbAndFLowInv` is a great name.

Something like `FinalizedChainIsPrefixOfAvailableChainInv` is probably better.
And a comment would help.